app/controllers/admin/authors_controller.rb

```diff
@@ -12,8 +12,11 @@ def update
     @author = Author.find(params[:id])
     @view = Admin::AuthorView.new(@author)
     outbound_lionpath_record = OutboundLionPathRecord.new(submission: @author.submissions.last, original_alternate_email: @author.alternate_email_address)
-    @author.attributes = author_params
+    author_attrs = author_params.merge(admin_edited_at: DateTime.now)
+    @author.attributes = author_attrs
     @author.save(validate: false)
+    # Update each submissions' updated_at timestamp so Solr detects an update
+    @author.submissions.each { |s| s.update :updated_at, DateTime.now }
```

Comment:
@AndrewGearhart  I added the code to update submissions' updated_at timestamps when authors' metadata is updated by admins as a part of this PR.